atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -27,33 +27,31 @@ import { useAuth0 } from "../../auth/authContext";
 const CreateCrashRecord = ({ client }) => {
   const { user } = useAuth0();
 
+  const userEmail = user.email;
+
   const unitsInitialState = [
-    { unit_desc_id: 1, atd_fatality_count: 0, sus_serious_injry_cnt: 0 },
+    { unit_desc_id: 1, fatality_count: 0, sus_serious_injry_cnt: 0 },
   ];
 
   const formInitialState = {
```

Comment:
this can be moved out of the component since it's constant

Reply:
moving them out of the component caused a bug when adding new units to the form and basically messed up the units state array so i think these can stay here?